dll/win32/comdlg32/rsrc.rc

```diff
@@ -146,9 +146,6 @@
 #ifdef LANGUAGE_UK_UA
     #include "lang/cdlg_Uk.rc"
 #endif
-#ifdef LANGUAGE_ZH_TW
-    #include "lang/cdlg_Tw.rc"
-#endif
```

Comment:
Why is this line (and associated file) removed?

Reply:
It was a wrong way to perform rebase. Fixed by rebasing it properly.